exotations/solvers/aico/src/AICOsolver.cpp

```diff
@@ -82,7 +75,7 @@ namespace exotica
     myfile << "Control";
     for (auto s : taskNames)
       myfile << " " << s;
-    for (int t = 0; t < T + 1; t++)
+    for (int t = 0; t < T; t++)
```

Comment:
It would have caused a memory issue silently. It didn't cause any problems in practice as far as I can remember so I guess the code was not executed very often (or at all) since the bug was introduced.